repo/packages/A/artifactory/2/config.json

```diff
@@ -0,0 +1,146 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "description": "Artifactory service properties.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of this DC/OS service.",
+          "type": "string",
+          "default": "artifactory"
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to this instance.",
+          "type": "number",
+          "default": 2.0,
+          "minimum": 1.0
+        },
+        "mem": {
+          "description": "Memory to allocate to this instance.",
+          "type": "number",
+          "default": 2048.0,
+          "minimum": 1024.0
+        },
+        "nodes": {
+          "description": "Number of Artifactory instances. By default, this is 1.",
+          "type": "integer",
+          "default": 1
+        },
+        "licenses": {
+          "description": "License for Artifactory Pro or comma separated licenses for Artifactory High Availability.",
+          "type": "string"
+        },
+        "host-volume": {
+          "description": "The location of a volume on the host to be used for persisting Artifactory data.",
+          "type": "string",
+          "default": "/var/artifactory"
+        },
+        "database": {
+          "description": "Database properties.",
+          "type": "object",
+          "properties": {
+            "type": {
+              "description": "Database type.",
```

Comment:
Please add in brackets the possible database types (e.g. `Database type (can be "mysql", "postgres" or ...).`)